integration_tests/mockApis/departure.ts

```diff
@@ -1,9 +1,10 @@
-import { SuperAgentRequest } from 'superagent'
+import { SuperAgentRequest, Response } from 'superagent'
```

Comment:
super meh, could choose to import these by type

Reply:
Good catch, ta!